src/contracts/core/DelegationManager.sol

```diff
@@ -365,33 +365,41 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
 
     /// @notice Migrates an existing queued withdrawal from the StrategyManager contract to this contract.
```

Comment:
tiny nit: this should be updated to reflect the fact that this is now accepting an array, rather than singlet input.

Reply:
Fixed